doc/helm/user-guide.md

```diff
@@ -15,31 +16,6 @@ powered by Helm using tools and libraries provided by the Operator SDK.
 **Note**: This guide uses [minikube][minikube_tool] version v0.25.0+ as the
 local Kubernetes cluster and [quay.io][quay_link] for the public registry.
 
-## Install the Operator SDK CLI
```

Comment:
IMO, we should leave the `## Install the Operator SDK CLI` headers and move the new line "Follow the steps in the installation guide..." under the header, so that the link is more prominent.

I think I also like having the prereqs listed first so that users have to at least skim past that section before getting to the installation section. If we linked to the installation stuff before listing prereqs, users may not see the prereqs. The downside is that it moves the install link a little further down the page.